docs/02-get-started/03-deploy-expose-microservice.md

```diff
@@ -0,0 +1,242 @@
+
+<div tabs name="Create a Service" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: v1
+  kind: Service
+  metadata:
+    name: orders-service
+    namespace: default
+    labels:
+      app: orders-service
+      example: orders-service
+  spec:
+    type: ClusterIP
+    ports:
+      - name: http
+        port: 80
+        protocol: TCP
+        targetPort: 8080
+    selector:
+      app: orders-service
+      example: orders-service
+EOF
+```
+
+To check that the Service was created successfully, run: 
+```bash
+kubectl get service orders-service -o=jsonpath="{.metadata.uid}"
+```
+
+The operation was successful if the command returns the **uid** of your Service.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+As you've already [created the Kubernetes Service with the microservice](#deploy-the-microservice), skip this part.
+
+<!--
+//TODO: Functionality not added yet. Check with Hasselhoffs in a while.
+If you created the Service at the previous step while creating the Deployment, skip this section. Otherwise, you must now create the Service.
+
+1. From the left navigation, go to **Services**.
+2. Click on **Create Service +**.
+3. ...
+
+The operation was successful if ... .
+--->
+  </details>
+</div>
+
+## Expose the microservice
+
+We have created the Service. Let's now expose it outside the cluster.
+
+> **CAUTION:** Exposing a workload to the outside world is always a potential security vulnerability, so tread carefully. In a production environment, always [secure the workload](../03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md) you expose.
+
+To expose our microservice, we must create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) CR for it, just like when we [exposed our Function](02-deploy-expose-function.md#expose-the-function).
+
+<div tabs name="Expose the microservice" group="deploy-expose-microservice">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.kyma-project.io/v1alpha1
+kind: APIRule
+metadata:
+  name: orders-service
+  namespace: default
+  labels:
+#    app: orders-service
+#    example: orders-service
+spec:
+  service:
+    host: orders-service.$CLUSTER_DOMAIN
+    name: orders-service
+    port: 80
+  gateway: kyma-gateway.kyma-system.svc.cluster.local
+  rules:
+    - path: /.*
+      methods: ["GET","POST"]
+      accessStrategies:
+        - handler: noop
+      mutators: []
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In your Services's view, click on **Expose Service +**.
+2. Provide the **Name** (`hello-world`) and **Hostname** (`hello-world`) and click **Create**.
```

Comment:
same name as for the API rule exposing the Function is not accepted

![image](https://user-images.githubusercontent.com/76950046/133647229-11a1fd6b-c90a-4843-b19c-e0641fb72f5a.png)

Reply:
Which is correct. I forgot to change the values here 🤦

Reply:
Fixed in https://github.com/kyma-project/kyma/pull/12128